src/UI/class-network-admin-sites-list.php

```diff
@@ -21,6 +20,12 @@
  */
 final class Network_Admin_Sites_List {
 	const COLUMN_NAME = 'parsely-api-key';
+	/**
+	 * Instance of Parsely class.
+	 *
+	 * @var Parsely
+	 */
+	private $parsely;
```

Comment:
Not sure why it wasn't defined before. Since we were using this in the codebase so defined it.